### ISSUE TITLE
No meaningful test output in GitHub Actions

### ISSUE THREAD

**Issue by indygreg** [nextest-rs/nextest#430]
I'm running `cargo nextest run`  (0.9.32) in GitHub Actions and am not seeing any output.

Example output (from https://github.com/indygreg/PyOxidizer/runs/7601284001?check_suite_focus=true):

```
 Compiling python-oxidized-importer v0.8.0-pre (/home/runner/work/PyOxidizer/PyOxidizer/python-oxidized-importer)
    Finished test [unoptimized + debuginfo] target(s) in 1m 06s
  Executable unittests src/lib.rs (target/debug/deps/pyembed-417c4a3c332f6dec)
```

I figured this might be due to terminal interaction shenanigans in the context of CI runners. So I added `--success-output immediate-final` to try to get it to stream output. But even that records nothing in the CI job output!

I also noticed the behavior occurs on the GitHub Actions jobs for the official nextest repo! From https://github.com/nextest-rs/nextest/runs/7594796482?check_suite_focus=true we see:

```
Run actions-rs/cargo@v1
  with:
    command: local-nt
    args: run --no-default-features --profile ci
    use-cross: false
  env:
    RUSTFLAGS: -D warnings
    CARGO_TERM_COLOR: always
    CACHE_ON_FAILURE: false
    CARGO_INCREMENTAL: 0
/home/runner/.cargo/bin/cargo local-nt run --no-default-features --profile ci
   Compiling nextest-runner v0.22.0 (/home/runner/work/nextest/nextest/nextest-runner)
   Compiling cargo-nextest v0.9.32 (/home/runner/work/nextest/nextest/cargo-nextest)
    Finished dev [unoptimized + debuginfo] target(s) in 18.04s
     Running `target/debug/cargo-nextest nextest run --no-default-features --profile ci`
    Finished test [unoptimized + debuginfo] target(s) in 0.12s
```

Not even getting a status output of how many tests were executed seems very odd to me. This isn't expected behavior, right?

**Comment by sunshowers** [comment]
Ahh whoops, this looks like it's due to the indicatif library upgrade. Looking at it now.

**Comment by sunshowers** [comment]
Just released cargo-nextest 0.9.33 with the fix -- binaries should show up within 15-20 minutes. Thanks for reporting this!

**Comment by indygreg** [comment]
Ack. Thanks for the quick turnaround!

Terminals sure are fun, aren't they :p
